### PR TITLE
Handle MultimapKeysSideInput in State GetRequests

### LINKED ISSUE #31628: Map Side Input Keys

### PR DESCRIPTION
**Pull request by damondouglas**
This PR closes #31628 adding handling for MultimapKeysSideInput in State GetRequests. Since this surfaced in Java validation tests of `org.apache.beam.sdk.transforms.ViewTest`, validated on Prism via:

```
TEST=org.apache.beam.sdk.transforms.ViewTest
./gradlew :runners:portability:java:ulrLoopbackValidatesRunnerTests -PjobEndpoint=localhost:8073 --tests="$TEST"
```

This PR fixes the following ViewTest tests:
- testMapAsEntrySetSideInput
- testWindowedMultimapAsEntrySetSideInput
- testWindowedMapAsEntrySetSideInput
- testMultimapAsEntrySetSideInput

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [x] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

**Comment by damondouglas** [comment]
R: @lostluck

File: sdks/go/pkg/beam/runners/prism/internal/worker/worker_test.go [hunk 1]
```diff
@@ -18,7 +18,11 @@ package worker
 import (
 	"bytes"
 	"context"
+	"github.com/apache/beam/sdks/v2/go/pkg/beam/core/graph/coder"
```
Comment (lostluck) [inline sdks/go/pkg/beam/runners/prism/internal/worker/worker_test.go:21]: Something is weird about your import plugin. It starts a new group, in the middle of the standard lib imports...

**Review by lostluck — APPROVED** [review]
Approved after import fix.

File: sdks/go/pkg/beam/runners/prism/internal/worker/worker_test.go (hunk 1 above)
Comment (lostluck) [reply sdks/go/pkg/beam/runners/prism/internal/worker/worker_test.go:21]: My bad, it doesn't start a new group, and it does sort them, it just puts them not with the rest of the non-std lib imports.

Comment (damondouglas) [reply sdks/go/pkg/beam/runners/prism/internal/worker/worker_test.go:21]: Something is really weird. I'll just make sure to apply go fmt before PRs from now on and not rely on the IDE

**Pull request merged**